### DISCUSSION TITLE
[Attached message]in <div class="ui grid stackable container">

### DISCUSSION THREAD

**Discussion by apple-idn** [fomantic/Fomantic-UI#697] [Collections (form, grid, menu, message, table, ...)] [answered]
# Bug Report

## Steps to reproduce
1. create Attached message in div class="ui grid stackable container"
2.top and bottom message not fluid

## Expected result
<img width="1171" alt="Jepretan Layar 2019-04-26 pukul 12 44 03 AM" src="https://user-images.githubusercontent.com/33564576/56753563-5db2f580-67bd-11e9-9241-5ccf7e2e4298.png">

## Actual result
<img width="1436" alt="Jepretan Layar 2019-04-26 pukul 12 43 23 AM" src="https://user-images.githubusercontent.com/33564576/56753559-5ab80500-67bd-11e9-8089-2e089a1ed2a9.png">

## Testcase
https://jsfiddle.net/allwe/c0t5uy2p/

## Screenshot (when possible)
![]()

## Version
2.7.4

**Comment by lubber-de** [comment, marked as answer]
If you work with `stackable grid`, you have to wrap your content into `column` divs.
Doing so will keep your example work as expected.
See your adjusted jsfiddle here: https://jsfiddle.net/yufb5s4e/3/
